articles/storage/blobs/lifecycle-management-policy-configure.md

```diff
@@ -228,6 +228,9 @@ A lifecycle management policy must be read or written in full. Partial updates a
 > [!NOTE]
 > The delete action of a lifecycle management policy won't work with any blob in an immutable container. With an immutable policy, objects can be created and read, but not modified or deleted. For more information, see [Store business-critical blob data with immutable storage](./immutable-storage-overview.md).
 
+> [!NOTE]
+> Kindly Note that while configuring the life cycle management policy if you add blob type as append blob, in that case for applying rule on base blob last accessed days option would not be available as it is not supported for append blob.
```

Comment:
```suggestion

```
If this was supposed to be removed by the other PR here. https://github.com/jainsourabh123/azure-docs/pull/4 - let's get this merged, or I can commit the change here and then we can merge this PR.